uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Form/AbsenceType.tpl

```diff
@@ -320,14 +320,44 @@
                   return hours_unit_value;
                 }
 
+                /**
+                 * Supports value submission of all CRM selectors
+                 * when such selectors are disabled
+                 */
+                function supportValueSubmissionOfAllSelectorsIfDisabled () {
+                  var $selectors = $('select.crm-select2');
+
+                  $.each($selectors, function (index, select) {
+                    supportValueSubmissionOfSelectorIfDisabled($(select));
+                  });
+                }
+
+                /**
+                 * Adds <input type="hidden"> before disabled <select>
+                 * with the same, watches <select> value and sets to the <input>
+                 *
+                 * @param {jQueryElement} $select - the selector to be amended
+                 */
+                function supportValueSubmissionOfSelectorIfDisabled ($select) {
```

Comment:
All the function definitions should be after the init function(`document.ready` in this case). Please move the document.ready function to the beginning.